src/test/java/htsjdk/samtools/util/IoUtilTest.java

```diff
@@ -394,4 +397,27 @@ public void testAssertDirectoryIsWritableFile(final File file, final boolean wri
             }
         }
     }
+    @DataProvider
+    public Object[][] fofnData() throws IOException {
+        Path fofnPath1 = inMemoryfileSystem.getPath("Level1.fofn");
+        Files.copy(new File(TEST_DATA_DIR.getAbsolutePath(),"Level1.fofn").toPath(), fofnPath1);

```

Comment:
This should be `TEST_DATA_DIR.getPath().resolve("Level1.fofn")`

Reply:
TIL. thanks.